packages/framework/src/Model/Customer/User/CustomerUserUpdateDataFactory.php

```diff
@@ -132,7 +136,7 @@ protected function getAmendedBillingAddressDataByOrder(Order $order, BillingAddr
     {
         $billingAddressData = $this->billingAddressDataFactory->createFromBillingAddress($billingAddress);
 
-        if ($billingAddress->getStreet() === null) {
+        if (empty($billingAddress->getStreet())) {
```

Comment:
it is not allowed to use `empty`, see the failed standards :slightly_smiling_face: 
Is that change because of possible empty strings? Is it worth adding a migration that updates empty strings to null in the relevant columns? :thinking: